bundle/metadata/properties.yaml

```diff
@@ -0,0 +1,4 @@
+properties:
```

Comment:
It seems like `olm.maxOpenShiftVersion` is not the same as `com.redhat.openshift.versions: "v4.6-v4.8"`
I guess i missunderstood the docs https://docs.openshift.com/container-platform/4.8/operators/operator_sdk/osdk-working-bundle-images.html
I will play with this later